### ISSUE TITLE
pybigtools bigbed write appears to store chrom as "rest"

### ISSUE THREAD

**Issue by nvictus** [jackh726/bigtools#54]
```python
import pybigtools as pbt

f = pbt.open("/tmp/foo.bb", "w")
f.write(
    {"chr1": 1000, "chr2": 1000}, 
    [
        ("chr1", 0, 100, "foo\tbar\t3"), 
        ("chr2", 100, 200, "xxx\tyyy\t1.0"),
        ("chr2", 200, 300, "zzz\twww\t1.0")
    ]
)
f.close()

f = pbt.open("/tmp/foo.bb", "r")
list(f.records("chr2"))
```

```
[(100, 200, 'chr2'), (200, 300, 'chr2')]
```

```python
print(f.sql())
```

```
table bed3
"Simple bed"
(
    string chrom;        "Reference sequence chromosome or scaffold"
    uint   chromStart;   "Start position in chromosome"
    uint   chromEnd;     "End position in chromosome"
)
```

**Comment by jackh726** [comment]
Uh, that's weird.

**Comment by jackh726** [comment]
This is fixed in pybigtools v0.2.2.

**Comment by nvictus** [comment]
It works! Thank you!

However, the autoSql still claims it's a bed3. Does autoSql have a way to declare at least something like "bed3+"?

Beyond that, it would be nice to able to store a custom autoSql definition, which we can use in [bioframe](https://github.com/open2c/bioframe/blob/main/bioframe/io/fileops.py#L653-L671).

Since in bioframe we can already infer common bed schemas from a dataframe and map numpy/pandas dtypes to autoSql types, we should be able to generate valid autoSql schemas from an input dataframe. Being able to store those upon writing a bigBed would keep that information in-band (and we could use it to automatically parse fields on read).
